addons/spreadsheet/static/src/global_filters/helpers.js

```diff
@@ -144,8 +144,10 @@ export function getRelativeDateDomain(now, offset, rangeType, fieldName, fieldTy
  * @returns {Array<Object>}
  */
 export function dateOptions(type) {
-    if (type === "month") {
-        return monthsOptions;
+    if (type === "monthQuarter") {
+        return [...monthsOptions, ...getPeriodOptions(DateTime.local()).filter(({ id }) =>
+            FILTER_DATE_OPTION["quarter"].includes(id)
+        )];
     } else {
```

Comment:
Not needed to be put in a if / else case anymore when there is only ONE expected `type`.  Crosscheck it's functionality once and update the `param` accordingly.